todoapp/todos/utils.py

```
@@ -1,4 +1,16 @@
 import json
+
+from django.contrib.postgres.aggregates import ArrayAgg
+from django.db.models import Count, Prefetch, Q
+from rest_framework import serializers
```

Ruff (0.12.2)

`rest_framework.serializers` imported but unused

Remove unused import: `rest_framework.serializers`

(F401)